ddlmaker.go

```diff
@@ -25,22 +25,22 @@ type DDLMaker struct {
 	Tables  []dialect.Table
 }
 
-// NewMaker return DDLMaker
-func NewMaker(conf Config) (*DDLMaker, error) {
-	dialect, err := dialect.NewDialect(conf.DB.Driver, conf.DB.Engine, conf.DB.Charset)
+// New creates a DDLMaker and returns it.
+func New(conf Config) (*DDLMaker, error) {
+	d, err := dialect.New(conf.DB.Driver, conf.DB.Engine, conf.DB.Charset)
 	if err != nil {
-		return nil, errors.Wrap(err, "error NewDialect()")
+		return nil, errors.Wrap(err, "error dialect.New()")
 	}
 
 	return &DDLMaker{
 		config:  conf,
-		Dialect: dialect,
+		Dialect: d,
 	}, nil
 }
 
 // AddStruct XXX
 func (dm *DDLMaker) AddStruct(ss ...interface{}) error {
-	pkgs := make(map[string]bool, 0)
+	pkgs := make(map[string]bool)
```

Comment:
The second parameter of `make` can be omitted when it is equal to its default value (0).